Cargo.toml

```diff
@@ -23,3 +23,12 @@ clap = "3.0.0-beta.5"
 tracing = "0.1"
 tracing-subscriber = "0.2"
 rsexp-derive = { version = "0.2.2", path = "rsexp-derive" }
+criterion = "0.3"
+rand = "0.8.4"
```

Comment:
Would you mind making these two `dev-dependencies` rather than plain `dependencies`? This would avoid having to drag these when building code that depends on rsexp.

Reply:
I think they might already be under `dev-dependencies`. Added `rand_pcg` below fyi.

Reply:
Oh you're right, I'll blame it on github review UI not giving enough context :)